packages/jobs/lib/runner/runner.ts

```diff
@@ -48,7 +50,15 @@ export async function getOrStartRunner(runnerId: string): Promise<Runner> {
         } catch (err) {}
     }
     const isRender = process.env['IS_RENDER'] === 'true';
-    const runner = isRender ? await RenderRunner.getOrStart(runnerId) : await LocalRunner.getOrStart(runnerId);
+    let runner: Runner;
+    if (isEnterprise()) {
```

Comment:
I would check for the existence of the env var since this is the prerequesite condition to make remote runner work

Reply:
Which env variable?
- `NANGO_ENTERPRISE`? --> if that isn't set this will just go to false
- `RUNNER_SERVICE_URL` --> defaults to the kubernetes address https://github.com/NangoHQ/nango-helm-charts/blob/main/charts/nango/values.yaml#L41

Reply:
sorry I am talking about `RUNNER_SERVICE_URL`